src/components/Collapsible/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import { Flipped, Flipper } from 'react-flip-toolkit';
+import { Props } from './types';
+
+
+/*
+** Collapsible component uses CSS transitions to smoothly hide and show a block of content.
+*/
+function Collapsible({
+    className,
+    anchorContent,
+    isCollapsed = true,
+    children,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const [collapsed, setCollapsed] = React.useState(isCollapsed);
+    const wrapperClasses = clsx(
+        'brew-Collapsible',
+        className,
+    );
+    const contentClasses = clsx(
+        {['brew-Collapsible__content']: collapsed},
+        {['brew-Collapsible--open']: !collapsed},
+    );
+
+
+    function toggleContent() {
+        setCollapsed(collapsed => !collapsed);
+    }
+
+
+    return (
+        <div className={wrapperClasses} {...otherProps}>
+            <Flipper flipKey={collapsed}>
```

Comment:
I like how you put the Flipper inside a wrapping div.  Maybe we should do the same thing for `Button` @lawren?  Makes it easier to target the element based on the wrapping className if you pass one.